Makefile

```diff
@@ -58,14 +58,15 @@ gh-preview html:
 
 gh-publish:
 	git checkout $(GH_PUBLISH_BRANCH)
-	git checkout $(GH_SOURCE_BRANCH) -- $(GH_SOURCE_DIR)
+	git rm -rf *
```

Comment:
We talked about this in person... adding `&& \ ` between each command will protect from unwanted behavior. The example we talked about being a failed checkout due to unstaged/uncomittted changes followed by a `git rm -rf *` despite the error. 

Looks good to me other than that!